mmv1/products/compute/TargetHttpsProxy.yaml

```diff
@@ -221,7 +221,7 @@ properties:
       load balancer (classic), this option is not available publicly.
   - !ruby/object:Api::Type::ResourceRef
     name: 'serverTlsPolicy'
-    resource: 'ServerSslPolicy'
+    resource: 'ServerTlsPolicy'
```

Comment:
I'm not sure I completely understand the rename here. Do you have any context to share with me?

Reply:
@ScottSuarez: this rename isn't actually necessary. I just noticed that it was called `ServerSslPolicy` but the field is actually server **TLS** policy and every other reference says TLS (not SSL) so I updated it, ran `make provider` and did not see a change in the code so I figured it was harmless. Is it? Found [this code](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/api/type.rb#L624), which talks about a reference to another resource but I'm not sure what is being referenced here. If you think it should stay as is, happy to revert this change.